src/UCommerce.Transactions.Payments.Braintree/BraintreePageBuilder.cs

```diff
@@ -121,6 +84,41 @@ private void AddPaymentForm(StringBuilder page, string paymentFormTemplate)
 					page.AppendLine(streamReader.ReadLine());
 			}
 		}
+		
+		/// <summary>
+        /// To use 3DS, you will need to pass an object with relevant customer and transaction data.
+		/// into the threeDSecure field of the requestPaymentMethod options in order to minimize the need
+		/// for issuing banks to present authentication challenges to customers.
+		///
+		/// This object must contain the `amount` field.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <param name="paymentRequest"></param>
+        private void AddThreeDSecureParameters(StringBuilder page, PaymentRequest paymentRequest)

```

Comment:
Just noticed the formatting is off in this case, I'll fix it together with few other minor things as part of another issue.